fable/CMakeLists.txt

```diff
@@ -40,9 +42,11 @@ set_target_properties(${target} PROPERTIES
     CXX_STANDARD_REQUIRED ON
     VERSION ${FABLE_VERSION}
 )
+configure_file(src/fable/version.hpp.in include/fable/version.hpp @ONLY)
 target_include_directories(${target}
   PUBLIC
     "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"
+    "$<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/include>"
```

Comment:
I don't think a generator expression is required here (same for line above)

Reply:
I think it's needed for this here at the bottom of the CMakeLists.txt file:

```cmake
install(EXPORT ${namespace}Targets
    NAMESPACE ${namespace}::
    DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/${namespace}
)
include(CMakePackageConfigHelpers)
write_basic_package_version_file(
    "${namespace}ConfigVersion.cmake"
    VERSION ${FABLE_VERSION}
    COMPATIBILITY AnyNewerVersion
)
install(FILES
    "cmake/${namespace}Config.cmake"
    "${CMAKE_CURRENT_BINARY_DIR}/${namespace}ConfigVersion.cmake"
    DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/${namespace}
)
```